docs/content/content/using-index-md.md

```diff
@@ -40,25 +40,31 @@ Now that you've got a handle on templates lets recap some Hugo basics to underst
 
 1. Sections and Taxonomies are 'List' pages, NOT single pages.
 2. List pages are rendered using the template heirarchy found in the [Content - List Template](http://localhost:1313/templates/list/) docs.
-3. The Home page is not technically a list page but has [it's own template](/templates/homepage/) at layouts/index.html.
+3. The Home page, though technically a List page, can have [it's own template](/templates/homepage/) at layouts/index.html rather than \_default/list.html. Many themes exploit this behaviour so you are likely to encounter this specific use case. 
 4. Taxonomy terms pages are 'lists of metadata' not lists of content, so [have their own templates](/templates/terms/). 
 
-Let's put this all together clearly because it's a source of much confusion:
+Let's put all this information together:
 
->**\_index.md files used in List pages, Terms pages or the Home page are NOT rendered as single pages or with Single Content templates, they are used to 'inject' frontmatter/content into the respective List pages, Terms pages or the Home page and those pages are rendered using templates listed above.**
+> **\_index.md files used in List pages, Terms pages or the Home page are NOT rendered as single pages or with Single Content templates.**
+
+> **All pages, including List pages, can have frontmatter and frontmatter can have markdown content - meaning \_index.md files are the way to _provide_ frontmatter and content to the respective List/Terms/Home page.**
 
 Here are a couple of examples to make it clearer...
 
-| \_index.md location               | Page affected             | Rendered by                   |
-| -------------------               | ------------              | -----------                   |
-| /content/post/\_index.md          | site.com/post/            | /layouts/section/post.html    |
-| /content/categories/hugo/index.md | site.com/categories/hugo/ | /layouts/taxonomy/hugo.html   |
+| \_index.md location                 | Page affected             | Rendered by                   |
+| -------------------                 | ------------              | -----------                   |
+| /content/post/\_index.md            | site.com/post/            | /layouts/section/post.html    |
+| /content/categories/hugo/\_index.md | site.com/categories/hugo/ | /layouts/taxonomy/hugo.html   |
+
+## Why \_index.md files are needed
```

Comment:
I think you should also remove this section, as it also implies "something magic" about these md files -- they are basically content files that must be named a special way so we can recognise them and so they do not interfere with other content files. The "need" should be obvious.